src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission.go

```diff
@@ -0,0 +1,86 @@
+package brokers
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/redpanda"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newDecommissionBroker(fs afero.Fs) *cobra.Command {
+	var force bool
+	cmd := &cobra.Command{
+		Use:   "decommission [BROKER ID]",
+		Short: "Decommission the given broker",
+		Long: `Decommission the given broker.
+
+Decommissioning a broker removes it from the cluster.
+
+A decommission request is sent to every broker in the cluster, only the cluster
+leader handles the request.
+`,
```

Comment:
Can we add to this a few new lines:

```
//
// For safety on v22.x clusters, this command will not run if the requested
// broker is in maintenance mode. As of v23.x, Redpanda supports
// decommissioning a node that is currently in maintenance mode. If you are on
// a v22.x cluster and need to bypass the maintenance mode check (perhaps your
// cluster is unreachable), use the hidden --force flag.
```

It's a bit detailed, but this is the only thing describing our hidden force flag.